pkg/services/sqlstore/migrations/migrations.go

```diff
@@ -157,4 +157,17 @@ func addStarMigrations(mg *Migrator) {
 
 	mg.AddMigration("create star table", NewAddTableMigration(starV1))
 	mg.AddMigration("add unique index star.user_id_dashboard_id", NewAddIndexMigration(starV1, starV1.Indices[0]))
+	mg.AddMigration("Add column dashboard_uid in star", NewAddColumnMigration(starV1, &Column{
+		Name: "dashboard_uid", Type: DB_Text, Nullable: true,
+	}))
+	mg.AddMigration("Add column org_id in star", NewAddColumnMigration(starV1, &Column{
+		Name: "org_id", Type: DB_BigInt, Nullable: false, Default: "1",
+	}))
+	mg.AddMigration(
+		"Update dashboard_uid and org_id value in star",
+		NewRawSQLMigration("").
+			SQLite("UPDATE star SET dashboard_uid = (SELECT uid FROM dashboard WHERE dashboard.id = star.dashboard_id), org_id = (SELECT org_id FROM dashboard WHERE dashboard.id = star.dashboard_id);").
```

Comment:
to support backwards migratinos (and then later coming forwar) this logic needs to be move to a background startup/cleanup job like we have for:

https://github.com/grafana/grafana-enterprise/blob/v11.3.0/src/pkg/extensions/report/database/database_mig.go#L306

Reply:
Done!